gpudirect-tcpx/nccl-test.yaml

```diff
@@ -1,143 +1,90 @@
-apiVersion: v1
-kind: Service
```

Comment:
you no longer need the services?

Reply:
We still need the node name to run the nccl-test. The manifest used to configure the service, because it needs to ssh into the node to run some commands. And running a fixed service name is simpler than checking the node name every time and copy paste into the command. 

For now, there are some simpler workaround, which are updated in the user guide.